actix-http/src/body/body.rs

```diff
@@ -25,7 +26,7 @@ pub enum Body {
     Bytes(Bytes),
 
     /// Generic message body.
-    Message(Pin<Box<dyn MessageBody>>),
+    Message(BoxAnyBody),
```

Comment:
`Pin<Box<dyn MessageBody<Error = Error>>>` is the alternative and locks in that error type. This way, converting to and from StdError is shown.